cmd/dtnd/configuration.go

```diff
@@ -84,44 +84,66 @@ func parseListenPort(endpoint string) (port int, err error) {
 }
 
 // parseListen inspects a "listen" convergenceConf and returns a Convergable.
-func parseListen(conv convergenceConf, nodeId bundle.EndpointID) (cla.Convergable, discovery.DiscoveryMessage, error) {
+func parseListen(conv convergenceConf, nodeId bundle.EndpointID) (cla.Convergable, bundle.EndpointID, core.CLAType, discovery.DiscoveryMessage, error) {
+	log.WithFields(log.Fields{
+		"EndpointID": conv.Node,
+		"Endpoint":   conv.Endpoint,
+		"Protocol":   conv.Protocol,
+	}).Debug("Initialising convergence adaptor")
+
+	// if the user has configured an EndpointID for this convergence adaptor
+	if conv.Node != "" {
+		parsedId, err := bundle.NewEndpointID(conv.Node)
+		if err != nil {
```

Comment:
This error is just logged, but not reported back. If a user configures an EID, it should also work, imho.